client/src/components/TrailForm.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from "react"
+
+const TrailForm = (props) => {
+  const [trailRecord, setTrailRecord] = useState({
+    trailName: "",
+    trailLength: "",
+    trailLocation: "",
+    trailDescription: "",
+    estimateTime: "",
```

Comment:
Didn't notice this until now, but your naming convention is very repetitive. If we are dealing with a `trail` resource, we don't need to specify `trailName` as the name because then we would be calling `trail.trailName`, where instead we want the _name_ for the _trail_ `trail.name`. Looking through your commit history I see that the creation of your `trail` migration was never put up for PR and was instead merged on your initial commit, so we didn't catch it initially. 

This is a conversation to have with your group before you make this change so that everyone is on the same page with updating the `trail` migration and other files involving trails going forward!

Reply:
Viji says he will handle this tomorrow morning with a new migration. But since the code is on my local environment, and I'll be absent tomorrow morning (dentist). Can we let this slide for this merge?

Reply:
All names are updated, using a new migration. trailName went south for the winter, only name shall remain.